core/extension/extension_api_dump.cpp

```diff
@@ -849,6 +850,22 @@ Dictionary NativeExtensionAPIDump::generate_extension_api() {
 			native_structures.push_back(d);
 		}
 
+		// TextServer structures
+		{
+			Dictionary d;
+			d["name"] = "Glyph";
+			d["format"] = "int start,int end,uint8_t count,uint8_t repeat,uint16_t flags,float x_off,float y_off,float advance,RID font_rid,int font_size,int32_t index";
+
+			native_structures.push_back(d);
+		}
+		{
+			Dictionary d;
+			d["name"] = "CaretInfo";
+			d["format"] = "Rect2 leading_caret,Rect2 trailing_caret,TextServer::Direction leading_direction,TextServer::Direction trailing_direction";
+
+			native_structures.push_back(d);
+		}
+
```

Comment:
I guess we should eventually move these definitions to the text server header as well, no sure how exactly. And do the same for `AudioFrame` and `GDVIRTUAL_NATIVE_PTR(AudioFrame)`